changelog.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+### Added 
+- Markdown pages for new InDesign UXP Features (Events, Menus, Script Result, DOM Versioning). (#50)
+- High resolution images for the guides/getting-started page. (#50)
```

Comment:
Keep in mind that the changelog is meant for third-party developers. Provide info that you think is relevant for them. Focus on features and refrain from listing house-keeping activities such as indentation, image resolution etc

Reply:
Done